src/OpenTelemetry/CompositeProcessor.cs

```diff
@@ -95,37 +95,28 @@ public override void OnStart(T data)
         /// <inheritdoc/>
         protected override bool OnForceFlush(int timeoutMilliseconds)
         {
+            var result = true;
             var cur = this.head;
-
             var sw = Stopwatch.StartNew();
 
             while (cur != null)
             {
                 if (timeoutMilliseconds == Timeout.Infinite)
                 {
-                    _ = cur.Value.ForceFlush(Timeout.Infinite);
+                    result = cur.Value.ForceFlush(Timeout.Infinite) && result;
```

Comment:
nit: can use `&=` operator

Reply:
I think no, `&=` has different semantic (short-circuit `ForceFlush` when `result` is true) than what we need (`ForceFlush` would always be called).